### PR TITLE
Remove lib/color.rb

### PR DESCRIPTION
**Pull request by duduribeiro**
Extracted from #1396

 In order to remove the lib folder from autoload, this commit removes the
 lib/color file, since it was not being used in any place. Only one test
 was using some of its constant.

### CODE REVIEW

File: features/support/hooks.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
-require 'color'
+BOLD  = "\033[1m"
```
Comment (hallelujah) [inline features/support/hooks.rb]: We should just use the colorize gem ....

Comment (duduribeiro) [reply features/support/hooks.rb]: installing one gem just for this case? isn't this too much? 🤔

Comment (Martouta) [reply features/support/hooks.rb]: i don't know that gem, but if u end up installing it, do it only for the test environment

Comment (hallelujah) [reply features/support/hooks.rb]: it is already installed `bundle show colorize`

File: features/support/hooks.rb [hunk 2]
```diff
@@ -188,13 +188,13 @@
...
-print_banner = -> (title, step) do
+print_banner = ->(title, step) do
   step_name = (step.try(:actual_keyword) || step.keyword) + step.name
-  Rails.logger.info <<-NEXT
+  Rails.logger.info <<~NEXT
 
-| #{title}: #{Color::BOLD}#{step_name}#{Color::CLEAR} |
-| #{'=' * (step_name.length + title.length + 2)} |
-#{step.multiline_arg}
+    | #{title}: #{step_name.colorize(:bold)} |
```
Comment (hallelujah) [inline features/support/hooks.rb]:
```suggestion
    | #{title}: #{step_name.bold} |
```

Comment (duduribeiro) [reply features/support/hooks.rb]: @hallelujah updated! 

thanks

**Pull request merged**